socorro/external/elasticsearch/supersearch.py

```diff
@@ -195,6 +195,44 @@ def get(self, **kwargs):
 
         search = search.filter(filters)
 
+        # Sorting.
+        sort_fields = []
+        for param in params['_sort']:
+            for value in param.value:
+                if not value:
+                    continue
+
+                # Values starting with a '-' are sorted in descending order.
+                # In order to retrieve the database name of the field, we
+                # must first remove the '-' part and add it back later.
+                desc = False
+                if value.startswith('-'):
```

Comment:
`.startswith()` only works on strings - is it possible that `value` might be an integer, for example?